<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1115</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1115</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1115</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1115</width>
     <height>300</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_Song{
	border-image: url(:/prefijoNuevo/Spotify_visualizer_logo.png);
	border-top-right-radius: 20px;
	border-bottom-right-radius: 20px;
}

#widget_AlbumCover{
	
	border-image: url(:/prefijoNuevo/Spotify_visualizer_logo.png);
	border-radius: 20px;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter">
     <widget class="QWidget" name="widget_AlbumCover" native="true">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget_AlbumCover{
	border-image: url(:/prefijoNuevo/Spotify_visualizer_logo.png);
}
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignVCenter">
     <widget class="QWidget" name="widget_Song" native="true">
      <property name="minimumSize">
       <size>
        <width>815</width>
        <height>210</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>0</width>
        <height>210</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget_Song{
	border-image: url(:/prefijoNuevo/Spotify_visualizer_logo.png);
}

</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_InfoSong" native="true">
         <property name="styleSheet">
          <string notr="true">#widget_InfoSong{
	/*background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.499963, fy:0.506, stop:0 rgba(1, 1, 1, 173), stop:1 rgba(111, 111, 111, 255));*/
	
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.499963, fy:0.506, stop:0 rgba(1, 1, 1, 173), stop:1 rgba(0, 0, 0, 255));
	border-top-right-radius: 20px;
	border-bottom-right-radius: 20px;
}

#label_SongName{
	font-family: Verdana;
	font-size: 30px;
	font-style: italic;
	font-weight: bold;
}

#label_SongArtist{
	font-family: Verdana;
	font-size: 15px;
	font-weight: bold;	
}


</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Policy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>62</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignBottom">
              <widget class="QLabel" name="label_SongName">
               <property name="minimumSize">
                <size>
                 <width>750</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>750</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Policy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>62</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item alignment="Qt::AlignmentFlag::AlignTop">
              <widget class="QLabel" name="label_SongArtist">
               <property name="minimumSize">
                <size>
                 <width>750</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../../../../../Descargas/res.qrc"/>
 </resources>
 <connections/>
</ui>
